benchmark/benchmark-normalization.hs

```diff
@@ -1,21 +1,30 @@
+{-# LANGUAGE CPP #-}
 {-# LANGUAGE PartialTypeSignatures #-}
 
-{-# OPTIONS_GHC -Wno-orphans                 #-}
+{-# OPTIONS_GHC -Wno-orphans #-}
```

Comment:
Do we still need this?

Reply:
Yes, this module has an orphan `NFData` instance for `Supply` which I took out at one point (and added back later)